whylabs_toolkit/monitor/models/analyzer/analyzer.py

```diff
@@ -57,8 +60,8 @@ class Analyzer(NoExtrasBaseModel):
     ] = Field(  # noqa F722
         None, description="A list of tags that are associated with the analyzer."
     )
-    # disabling CronSchedule as it can be tricky on the BE
-    schedule: Optional[FixedCadenceSchedule] = Field(  # Optional[Union[CronSchedule, FixedCadenceSchedule]] = Field(
+
+    schedule: Optional[Union[FixedCadenceSchedule, CronSchedule]] = Field(
```

Comment:
Are there any sideeffects to setting the schedule to None like this? In particular it would be better if the field wasnt included in a generated yaml as we dont particularly want people to start experimenting with it

Reply:
by default we'll fill in with a `FixedCadenceSchedule` if the user doesn't set anything in the toolkit. if they explicitly unset the schedule, the analyzer is considered disabled by the monitor. do you think we should not allow it at all?

Reply:
its fine if we normally default it to fixed. I forgot it's always expected